app/Http/routes.php

```diff
@@ -36,6 +36,14 @@
 | Logging In/Out Routes
 |--------------------------------------------------------------------------
 */
-Route::get('/auth/login', 'Auth\AuthController@getLogin');
-Route::post('/auth/login', 'Auth\AuthController@postLogin');
-Route::get('/auth/logout', 'Auth\AuthController@getLogout');
+$router->group([
+    'namespace' => 'Auth',
+    'prefix' => 'auth',
+], function(){
+    Route::get('login', 'AuthController@getLogin');
+    Route::post('login', 'AuthController@postLogin');
+    Route::get('logout', 'AuthController@getLogout');
+    Route::match(['get', 'post'], 'register', function(){
+        return redirect('/');
```

Comment:
You could change the trait user by AuthController to AuthenticatesUsers and get rid of this redirect